library/CMakeLists.txt

```diff
@@ -81,6 +81,8 @@ set(F3D_PUBLIC_HEADERS
   ${CMAKE_CURRENT_SOURCE_DIR}/public/loader.h
   ${CMAKE_CURRENT_SOURCE_DIR}/public/log.h
   ${CMAKE_CURRENT_SOURCE_DIR}/public/options.h
+  ${CMAKE_CURRENT_SOURCE_DIR}/public/reader.h
+  ${CMAKE_CURRENT_SOURCE_DIR}/public/plugin.h
```

Comment:
Did you check that the doxygen on this file is correct ? 

Also the libf3d doc needs updating.

Reply:
Good point, I'll check